lib/src/index.ts

```diff
@@ -5,4 +5,5 @@
  */
 export * from './cmcd.js';
 export * from './id3.js';
+export * from './request.js';
```

Comment:
This line breaks the tests due to a quirk of the build system. The `request` package only exports types/interfaces, and as a result when the Typescript is compiled, there is no `request.js` file. To get around this we need to add the `type` modifier:

```suggestion
export type * from './request.js';
```

Reply:
Fixed